README.md

```diff
@@ -279,7 +321,7 @@ Any additional props are passed to the pressable component.
 |inactiveColor|`string \| undefined`|`null`|Color applied to the label when inactive|
 |inactiveOpacity|`number \| undefined`|`0.7`||
 |index|`number`|||
-|indexDecimal|`SharedValue<number>`|||
+|indexDecimal|`any`|||
```

Comment:
Seems like there's a problem here 🤔

Reply:
I was running the doc script with a different reanimated version. This is weird, but re-running with the rc.2 fixed